drracket/drracket/private/language.rkt

```diff
@@ -350,27 +348,25 @@
   ;; simple-module-based-language-render-value/format : TST settings port (union #f (snip% -> void)) (union 'infinity number) -> void
   (define (simple-module-based-language-render-value/format value settings port width)
     (let-values ([(converted-value write?)
-                  (call-with-values
-                      (lambda ()
-                        (simple-module-based-language-convert-value value settings))
-                    (case-lambda
-                     [(converted-value) (values converted-value #t)]
-                     [(converted-value write?) (values converted-value write?)]))])
-      (let ([pretty-out (if write? pretty-write pretty-print)])
-        (setup-printing-parameters
-         (λ ()
-            (cond
-             [(simple-settings-insert-newlines settings)
-              (if (number? width)
-                  (parameterize ([pretty-print-columns width])
-                    (pretty-out converted-value port))
-                  (pretty-out converted-value port))]
-             [else
-              (parameterize ([pretty-print-columns 'infinity])
-                (pretty-out converted-value port))
-              (newline port)]))
-         settings
-         width))))
+                  (call-with-values (lambda ()
+                                      (simple-module-based-language-convert-value value settings))
+                                    (case-lambda
+                                      [(converted-value) (values converted-value #t)]
+                                      [(converted-value write?) (values converted-value write?)]))])
+      (define pretty-out (if write? pretty-write pretty-print))
+      (setup-printing-parameters (λ ()
+                                   (cond
```

Comment:
Ditto here.

Reply:
Here the Resyntax fix I mentioned above isn't workable, since this code needs to be reformatted to correct its leading indentation. So this is a `fmt` issue. @sorawee, this increase in indentation seems quite unfortunate.